Marlin/Conditionals_post.h

```diff
@@ -780,11 +780,11 @@
   #endif
 
   #if IS_KINEMATIC
-    // Check for this in the code instead
-    #define MIN_PROBE_X X_MIN_POS
-    #define MAX_PROBE_X X_MAX_POS
-    #define MIN_PROBE_Y Y_MIN_POS
-    #define MAX_PROBE_Y Y_MAX_POS
+    // Check for this in the code instead (preprocessor can't do floating point arithmetic)
+    #define MIN_PROBE_X (-10000)
+    #define MAX_PROBE_X ( 10000)
+    #define MIN_PROBE_Y (-10000)
+    #define MAX_PROBE_Y ( 10000)
```

Comment:
I changed the delta MIN_PROBE_X etc defines to be PRINTABLE_RADIUS based again, and removed the sanity checks for range-checking the UBL_PROBE_PT values for DELTA.  Hopefully this better reflects reality.